docs-chef-io/content/server/upgrade_server.md

```diff
@@ -14,43 +14,37 @@ aliases = ["/upgrade_server.html", "/upgrade_server/"]
     weight = 60
 +++
 
-## Chef Infra Server Upgrade Matrix
+Each new release of Chef Infra Server improves reliability and updates 3rd party components to ensure the security of the server. It is important to keep Chef Infra Server up to date in order to ensure the secure and reliable operation of Chef Infra in your organization.
 
-| Running Version | Upgrade Version | License | Version Support |
+{{< warning >}}
+Before upgrading a production server make sure to upgrade a test server to confirm the process.
+{{< /warning >}}
+
+## Upgrade Matrix
+
+If running a Chef Infra Server 12.17.15 or later you can upgrade directly to the latest releases of Chef Infra Server 14. If you are running a release prior to 12.17.15 you must perform a stepped upgrade as outlined below.
+
+| Running Version | Upgrade To Version | Requires License | Supported Version |
 |---------|---------|------|-----------|
 | 13 | 14 | Yes | Yes |
-| 12.17.15 | 14 | Yes | Yes |
+| 12.17.15 | 14 | Yes | No |
 | 12.3.0 | 12.17.15 | No | No |
 | 11 | 12.3.0 | No | No |
 
-License
-: Chef Infra Server 13 and 14 are governed by the [Chef License]({{< relref "chef_license_accept#chef-infra-server" >}}). You will be required to accept these terms when using Chef Infra Server 13 or 14 for the first time by entering `Yes` when prompted.
-
-Version Support
-: Chef Infra Server 13 and 14 are supported Chef Software distributions. Earlier versions are no longer supported. For more information about supported Chef Software see the [Supported Versions]({{< relref "versions#supported-commercial-distributions" >}}) documentation.
-
-## Upgrading to Chef Infra Server
-
-Three upgrade scenarios exist for upgrades from Chef Infra Server 12.17.15 to Chef Infra Server 13 or 14:
+Requires License
+: Chef Infra Server 13 and later are governed by the [Chef EULA](/chef_license). You will be required to accept these terms when using Chef Infra Server for the first time by entering `Yes` when prompted.
 
-- [Standalone]({{< relref "#standalone-upgrade" >}})
-- [Chef Backend (HA)]({{< relref "#chef-backend-upgrade" >}})
-- [Tiered]({{< relref "#tiered-upgrade" >}})
+Supported Release
+: Chef Infra Server 13 and later are currently supported Chef Software releases. Earlier releases are no longer supported as of 12/31/2020. For more information about supported Chef Software see the [Supported Versions](https://docs.chef.io/versions/#supported-commercial-distributions) documentation.
 
-### Chef Infra Server 14
+### Upgrading to 14.x
 
-{{< warning >}}
-
-Do not upgrade your production server. First, upgrade in your test server, and then upgrade your production server.
-
-{{< /warning >}}
-
-Chef Infra Server 14 uses Elasticsearch as its search index.
+Chef Infra Server 14 moved from Solr to Elasticsearch as its search index.
 {{% server_upgrade_duration %}}
```

Comment:
I see what server_upgrade_duration translates into in the rendered page; but cannot seem to find where it lives.

Reply:
It is a shortcode (a partial) so it is in the main repo, which makes it available to all of the docs sets. We pull the partials in through the _vendor files, which can't be edited in this repo. 

The file is at: https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/server_upgrade_duration.md

You can preview the content at https://deploy-preview-2208--chef-server.netlify.app/upgrade_server

Reply:
@PrajaktaPurohit Do you have any concerns about this content, or can I resolve this issue?